include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -49,56 +49,12 @@ DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
 #endif
 
 #include <fstream>
+#include <vector>
 
 
 DEAL_II_NAMESPACE_OPEN
 
 
-namespace internal
```

Comment:
@kronbichler Do you see any issues in this file?

Reply:
That will be further cleaned up with #10334 once this is done, so fine here :+1: